index.js

```diff
@@ -166,13 +172,15 @@ Parser.prototype._emit = function (Row, cells) {
 }
 
 Parser.prototype._oncell = function (buf, start, end) {
-  if (buf[start] === this.escape && buf[end - 1] === this.escape) {
+  // remove quotes from quoted cells
+  if (buf[start] === this.quote && buf[end - 1] === this.quote) {
     start++
     end--
   }
 
   for (var i = start, y = start; i < end; i++) {
-    if (buf[i] === this.escape && buf[i + 1] === this.escape) i++
+    // check for escape characters and skip them
+    if (buf[i] === this.escape && buf[i + 1] === this.quote) i++
```

Comment:
i guess there should be a bound check here as well. not sure why we didn't have that before